lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -1649,7 +1651,7 @@ def main():
             customize=dict(required=False, type='bool', default=False),
             dns_servers=dict(required=False, type='list', default=None),
             domain=dict(required=False, type='str', default=None),
-            networks=dict(required=False, type='dict', default={}),
+            networks=dict(required=False, type='dict', default=[]),
```

Comment:
Ah, good catch. You won't believe it, but the whole patch was typed from one isolated system to my laptop :-) Don't ask...